xmipp

```diff
@@ -277,10 +277,13 @@ def pull():
 def cloneOrCheckout(repo, branch):
     repo_dir = os.path.join('src', repo) if repo != XMIPP else '.'
     if branch not in getAllBranchesAndTags(repo)[1]:
-        branch = getBranch(repo, repo_dir)[1]
+        branch = None
 
     if not os.path.exists(repo_dir):
-        print('Clonning repository: {}/{}'.format(repo, branch))
+        if branch == None:
```

Comment:
maybe instead of:
```Python
if branch not in getAllBranchesAndTags(repo)[1]:
    branch = None
 if not os.path.exists(repo_dir):
        if branch == None:
            branchName = 'HEAD'
```
It's simpler:
```Python
if not os.path.exists(repo_dir):
        if branch not in getAllBranchesAndTags(repo):
             branchName = 'HEAD'
```